persistence/src/main/scala/hmda/persistence/institutions/InstitutionPersistence.scala

```diff
@@ -12,6 +12,10 @@ object InstitutionPersistence {
 
   val name = "institutions"
 
+  case class GetInstitutionById(institutionId: String) extends Command
+  case class GetInstitutionsById(ids: List[String]) extends Command
+  case class FindInstitutionByDomain(domain: String) extends Command
```

Comment:
These messages don't have `protobuf` converter / serialization code associated with them. They will be sent as `JavaSerializable` which is slow. It would be nice to include this code as part of this PR

Reply:
RIght, forgot about those!  I'll post an update soon.